arcgis10_mapping_tools/RenameLayer/RenameLayer/ConstructLayerName.cs

```diff
@@ -109,114 +107,11 @@ public static class RenameLayerToolValues
         public static Dictionary<string, string> csvToDictionary(string @path)
         {

```

Comment:
I am concerned as to what's happening here. So far as I can see there are quite a number of changes made to the RenameLayerTool code on the master branch - this method is one substantive example which on the master branch is rewritten for parsing CSV files more robustly. It looks to me like merging MapTemplateTests branch would overwrite those changes which I doubt is desired. 

If this analysis is right, then we might need to tease out renamelayertool changes into a separate branch, or, discard the later "changes" on this branch that would overwrite them?